dev-python/exdown/exdown-0.7.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit distutils-r1 virtualx
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/nschloe/${PN}.git git://github.com/nschloe/${PN}.git"
+else
+	SRC_URI="https://github.com/nschloe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Extract code blocks from markdown"
+HOMEPAGE="https://github.com/nschloe/exdown"
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="
+	$( python_gen_cond_dep \
+		'dev-python/importlib_metadata[${PYTHON_USEDEP}]' python3_7 )
+"
```

Comment:
Can you explain to me what this does, and how it interacts with *distutils-r1.eclass*? Should you rather choose *python-single-r1.eclass* or use `DISTUTILS_SINGLE_IMP`?

Reply:
This depends on importlib_metadata for python versions lower than python3.8, for py3.8 and up the dependency is not needed.

Reply:
Thanks for explaining, haven't seen this syntax being used before :)